article/articles.py

```diff
@@ -2,14 +2,30 @@
 
 from directory_constants.constants import exred_articles
 from django.urls import reverse_lazy
+from django.utils.functional import cached_property
+
+from . import helpers
 
-Article = namedtuple(
-    'Article',
-    ['uuid', 'title', 'keywords', 'tasks', 'markdown_file_path', 'url',
-     'parent']
-)
 ArticleParent = namedtuple('ArticleParent', ['uuid', 'title'])
 
+
+class Article:
+
+    def __init__(self, uuid, title, keywords, tasks,
+                 markdown_file_path, url, parent):
+        self.uuid = uuid
+        self.title = title
+        self.keywords = keywords
+        self.tasks = tasks
+        self.markdown_file_path = markdown_file_path
+        self.url = url
+        self.parent = parent
+
+    @cached_property
+    def time_to_read(self):
+        return helpers.time_to_read_in_minutes(self)
```

Comment:
why not just `time_to_read = helpers.time_to_read_in_minutes(self)` in `__init__`?

Reply:
Theoretically, the advantage of `cached_property` over the `__init__` is to avoid that the `time_to_read` value is accidentally set to something else. `cached_property` would create a property that cannot be changed. I think in this case is not very important. @konradko

Reply:
Another reason to use `cached_property`: doing `time_to_read = helpers.time_to_read_in_minutes(self`) in `__init__` produces a circular import (thanks @richtier for debugging that). So I'll keep this as a property, it's cached anyway.

Reply:
Resolving the circular import is quite a big task and, for the reason I said above, I'd prefer to use `cached_property`. I created a dedicated ticket for the circular import https://uktrade.atlassian.net/browse/ED-2475